### PR TITLE
Updated the simulator creation to include an evvVar called NAMESPACE

### PR DESCRIPTION
**Pull request by SeanCondon**

### CODE REVIEW

File: pkg/onit/cluster/simulator.go [hunk 1]
```diff
@@ -359,6 +361,15 @@ func (s *Simulator) createPod() error {
 							Name:  gnmiInsecurePortEnv,
 							Value: fmt.Sprintf("%d", s.Port()),
 						},
+						{
+							Name: kube.NamespaceEnv,
```
Comment (adibrastegarnia) [inline pkg/onit/cluster/simulator.go]: It looks like you defined namespaceEnv as constant but you used something else. Am I missing something?

Comment (kuujo) [reply pkg/onit/cluster/simulator.go]: That's the name of the environment variable (`NAMESPACE`), and the value is taken from the k8s configuration

**Comment by kuujo** [comment]
Linter error

File: pkg/onit/cluster/simulator.go (hunk 1 above)
Comment (kuujo) [reply pkg/onit/cluster/simulator.go]: Sorry I thought you meant the variable used here :-)

Comment (SeanCondon) [reply pkg/onit/cluster/simulator.go]: yeah - i had defined it, and then saw it had already been defined, and forgot to remove it. Removed now.

**Pull request merged**